components/lcd_touch/esp_lcd_touch_gt911/esp_lcd_touch_gt911.c

```diff
@@ -132,7 +180,17 @@ static esp_err_t esp_lcd_touch_gt911_read_data(esp_lcd_touch_handle_t tp)
 
     /* Any touch data? */
     if ((buf[0] & 0x80) == 0x00) {
+#if (CONFIG_ESP_LCD_TOUCH_MAX_BUTTONS > 0)
+        tp->data.button[0].status = 0x00;
```

Comment:
How many buttons are supported in GT911. Should it be cleared all supported buttons?

Reply:
> How many buttons are supported in GT911. Should it be cleared all supported buttons?

According to the screenshots shown above, it seems that I can only get one event.